addons/assistance.py

```diff
@@ -445,5 +445,17 @@ def check_console(self, message, channel, consoles):
         """Erase pending updates on Nintendo Switch"""
         await self.simple_embed("When an update is downloaded, but not installed, the console will not display the firmware version in System Settings. \n\n• To fix this, *power the console off* (hold the power button, follow on-screen prompts). ***Hold*** Volume Down and Volume Up, then Power. When you see Maintenance Mode, you may reboot, and check System Settings. \n\n *To block automatic update downloads, enter 104.236.106.125 as your primary DNS and 62.210.147.20 as your secondary DNS for your home network.*", title="How to delete pending Switch Updates")
 
+    # Embed to xGhostBoysx RCM guide
+    @commands.command()
+    @commands.cooldown(rate=1, per=5.0, type=commands.BucketType.channel)
```

Comment:
why 5.0 instead of 30.0?

Reply:
30seconds might be a bit long, if switch assistance takes off